aip/general/0193.md

```diff
@@ -47,19 +47,31 @@ following:
     "details": [
       {
         "@type": "type.googleapis.com/google.rpc.ErrorInfo",
-        "reason": "STOCKOUT",
+        "reason": "resource_availability",
         "domain": "compute.googleapis.com",
         "metadata": {
-          "zone": "us-east1-a",
           "vmType": "e2-medium",
           "attachment": "local-ssd=3,nvidia-t4=2"
+          "zone": "us-east1-a",
          }
        },
        {
         "@type": "type.googleapis.com/google.rpc.LocalizedMessage",
         "locale": "en-US",
-        "message": "A e2-medium with local-ssd=3,nvidia-t4=2 is currently unavailable in the us-east1-a zone.\nCapacity changes frequently, so try your request in a different zone, with a different VM hardware configuration, or at a later time. For more information, see troubleshooting documentation."
+        "message": An <e2-medium> VM instance with <local-ssd=3,nvidia-t4=2> is currently unavailable in the <us-east1-a> zone.\n
```

Comment:
this field cannot contain line breaks, as it breaks json formatting. Please keep the content on a single line and use just escaped line breaks, '\n', instead.

Reply:
yes, thank you.